apps/api-gateway/src/credential-definition/credential-definition.controller.ts

```diff
@@ -76,45 +77,48 @@ export class CredentialDefinitionController {
     summary: 'Fetch all credential definitions of provided organization id with pagination',
     description: 'Fetch all credential definitions from metadata saved in database of provided organization id.'
```

Comment:
I think this message `from metadata` is misleading.

Reply:
Resolved